atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -373,6 +374,14 @@ def add_log_type(output):
         metadata_version = 0
 
         worker_metadatas = self.workflow.postbuild_results.get(PLUGIN_FETCH_WORKER_METADATA_KEY)
+        maven_components = get_maven_components(self.workflow)
+
+        # add maven components alongside RPM components
+        for platform in worker_metadatas.keys():
+            for component in worker_metadatas[platform]['output']:
```

Comment:
this isn't component but entries in output
so call it 'output'

for output in worker_metadatas[platform]['output']: